app/app/src/main/java/com/globant/calculator/android/mvp/presenter/CalculatorPresenter.java

```diff
@@ -1,54 +1,103 @@
 package com.globant.calculator.android.mvp.presenter;
 
+import android.icu.text.DecimalFormat;
+
 import com.globant.calculator.android.mvp.model.CalculatorModel;
 import com.globant.calculator.android.mvp.view.CalculatorView;
 
-import static com.globant.calculator.android.utils.Constants.EMPTY_STRING;
+import static com.globant.calculator.android.utils.Constants.DIVIDE;
+import static com.globant.calculator.android.utils.Constants.DOT_BUTTON;
+import static com.globant.calculator.android.utils.Constants.MINUS;
+import static com.globant.calculator.android.utils.Constants.MULTIPLY;
 import static com.globant.calculator.android.utils.Constants.NUMBER_ZERO;
-import static com.globant.calculator.android.utils.Constants.ZERO_INT;
+import static com.globant.calculator.android.utils.Constants.PLUS;
+import static com.globant.calculator.android.utils.Constants.ZERO_DOUBLE;
 
 public class CalculatorPresenter {
 
+    private static DecimalFormat df2 = new DecimalFormat("#.##");
```

Comment:
Is the static needed??
Please put a more representative name instead of df2
no hardcode pls, put "#.##" in a constante

Reply:
is only use there so i remove the static.
the other changes are done